components/org.wso2.carbon.identity.oauth.ui/src/main/java/org/wso2/carbon/identity/oauth/ui/client/OAuthAdminClient.java

```diff
@@ -126,4 +127,17 @@ public void updateOauthApplicationState(String consumerKey, String newState) thr
     public OAuthTokenExpiryTimeDTO getOAuthTokenExpiryTimeDTO() throws RemoteException {
         return stub.getTokenExpiryTimes();
     }
+
+    /**
+     * Get the registered scope validators from OAuth server configuration file
```

Comment:
Java-doc comment should end with a period. Could you please add a period(.) at the end of this comment?